pymeasure/instruments/attocube/adapters.py

```diff
@@ -22,10 +22,48 @@
 # THE SOFTWARE.
 #
 
+import re
 import time
 
 from pymeasure.adapters import TelnetAdapter
 
+# compiled regular expression for finding numerical values in reply strings
+reg_value = re.compile("\w+\s+=\s+([-+]?[0-9]*\.?[0-9]+)")
```

Comment:
You should use raw strings, prefixed with `r`, when feeding regular expressions to avoid intepreting the backslashes - `r'blabla'`

Reply:
Also, maybe make this variable _reg_value to signal that it should not be used by others.